ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -276,3 +277,11 @@ def greater_than(self, y, name=None):
     @with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
     def rsqrt(self, name=None):
         return ivy.reciprocal(ivy.sqrt(self._ivy_array))
+
+    @with_supported_dtypes(
+        {"2.5.0 and below": ("int32", "int64")},
+        "paddle",
+    )
+    def floor_divide(self, y, name=None):
+        y_ivy_array = _to_ivy_array(y)
+        return ivy.floor_divide(self._ivy_array, y_ivy_array)

```

Comment:
Here, instead you can pass in `y._ivy_array`. The `_ivy_array` attribute inside the `paddle.Tensor` objects already contains their `ivy.Array` equivalent so we don't need to explicitly convert `y` to an ivy array